pulp_ansible/app/models.py

```diff
@@ -372,6 +372,19 @@ class DownloadLog(BaseModel):
     )
 
 
+class CollectionDownloadCount(BaseModel):
```

Comment:
Is this orthogonal to the CollectionDownloadLog?

Reply:
I looked into using `CollectionDownloadLog` to base the data off of, but there were numerous complications that were eliminated with a separate `CollectionDownloadCount`.

The complications included counting objects in a log that is very similar to an access log (high activity on community Galaxy exacerbates this). That lead to caching strategies (always fun) and time stamp based download count updates via a `TaskSchedule` as well as log maintenance.

Reply:
Also on a high traffic system (multiple million downloads weekly), `CollectionDownloadLog` may need to have records deleted occasionally due to db growth. So yes they are orthogonal.